frontend/src/actionCreators/parcelsActionCreator.spec.tsx

```diff
@@ -27,9 +27,9 @@ describe('fetchParcels action creator', () => {
     const mockResponse = { data: { success: true } };
     mockAxios.onGet(url).reply(200, mockResponse);
     return fetchParcels(null)(dispatch).then(() => {
-      expect(requestSpy).toHaveBeenCalledTimes(0);
-      expect(successSpy).toHaveBeenCalledTimes(0);
-      expect(dispatch).toHaveBeenCalledTimes(0);
+      expect(requestSpy).toHaveBeenCalledTimes(1);
```

Comment:
I'm not sure this is correct.  I had to change them to 0 when `null` is passed to `fetchParcels(...)` for the test to pass.  But I could be doing something wrong.

Reply:
The two scenarios that we support are 
1) null boundaries
or
2) boundaries where the area is bigger then a point
So this should have a request, success, etc.